.eslintrc.js

```diff
@@ -1,57 +1,19 @@
 'use strict';
 
+const { configs } = require('@nullvoxpopuli/eslint-configs');
+
+// accommodates: JS, TS, App, and Addon
+const config = configs.ember();
+
 module.exports = {
-  globals: {
-    window: true,
-  },
-  root: true,
-  parser: 'babel-eslint',
-  parserOptions: {
-    ecmaVersion: 2018,
-    sourceType: 'module',
-    ecmaFeatures: {
-      legacyDecorators: true,
-    },
-  },
-  plugins: ['ember'],
-  extends: [
-    'eslint:recommended',
-    'plugin:ember/recommended',
-    'plugin:prettier/recommended',
-  ],
-  env: {
-    browser: true,
-  },
-  rules: {},
+  ...config,
   overrides: [
-    // node files
+    ...config.overrides,
     {
-      files: [
-        '.eslintrc.js',
-        '.prettierrc.js',
-        '.template-lintrc.js',
-        'ember-cli-build.js',
-        'index.js',
-        'testem.js',
-        'blueprints/*/index.js',
-        'config/**/*.js',
-        'tests/dummy/config/**/*.js',
-      ],
-      excludedFiles: [
-        'addon/**',
-        'addon-test-support/**',
-        'app/**',
-        'tests/dummy/app/**',
-      ],
-      parserOptions: {
-        sourceType: 'script',
-      },
-      env: {
-        browser: false,
-        node: true,
+      files: ['**/*.js', '**/*.ts'],
+      rules: {
+        'prettier/prettier': ['error', require('./.prettierrc')],
```

Comment:
the defaults from the lint meta-package have the printWidth set to 100, and that caused too many changes, so I figured it would be best to just use the existing prettier config